scripts/acbuild-rkt-builder.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -ex
+
+MODIFY=${MODIFY:-""}
+FLAGS=${FLAGS:-""}
+IMG=${IMG:-"debian"}
+IMGVERSION=${VERSION:-"sid"}
+DOCKERIMG="$IMG:$IMGVERSION"
+ACIFILE=${ACIFILE:-"$IMG-$IMGVERSION.aci"}
+OUTACI="rkt-builder.aci"
+BUILDDIR=/opt/build-rkt
+SRC_DIR=/opt/rkt
+ACI_GOPATH=/go
+VERSION=${VERSION:-"v0.10.0"}
```

Comment:
How about defaulting it to `v0.10.0+git`? That way our `bump-release` script will bump this version automatically.